lua/pac3/core/client/base_part.lua

```diff
@@ -53,6 +56,18 @@ function PART:IsValid()
 	return self.is_valid
 end
 
+local pac_enable_convars = {}
+local function get_enable_convar(classname)
+	local convar = pac_enable_convars[classname]
+
+	if not convar then
+		convar = GetConVar("pac_enable_" .. string.Replace(classname, " ", "_"):lower())
+		pac_enable_convars[classname] = convar
+	end
+
+	return convar
+end
```

Comment:
This can be the __index metamethod of pac_enable_convars. Then replace calls to this with table indexes.

Reply:
Speed doesn't matter for this function. it's just cleaner to use __index. And yeah jit is kinda random about what it blacklists or compiles